runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -75,9 +75,12 @@ protected void shutDown() throws Exception
 	@Subscribe
 	public void onVarbitChanged(VarbitChanged event)
 	{
-		favor = client.getVar(Varbits.KINGDOM_FAVOR);
-		coffer = client.getVar(Varbits.KINGDOM_COFFER);
-		processInfobox();
+		if(isInKingdom())
```

Comment:
this is
1) wrong
2) saving zero time

Reply:
Hmm yea it is blatantly wrong and calls IsInKingdom twice. I think profiling via instrumentation inflated the cost of getting varbits and made it appear like that was superexpensive. My bad